### DISCUSSION TITLE
Is there a way to change the default icon set on the button itself?

### DISCUSSION THREAD

**Discussion by iamritiksharma** [add2cal/add-to-calendar-button#624] [Q&A] [answered]
I tried many things to change the default icon used for the button itself but no effort was fruitful.
Things I tried:
- Using ::before in the css styling. I want to use the font awesome icon but it seems like the  font awesome stylesheet isn't loaded in shadow root.
- Accessing the shadow root elements using document object and trying to add it that way.
- Using custom styling. It also removes all the default styling which I still want to use in the button and the calendar list items.
- Using atcb_actions. It also works but only opens the options in a modal but I want it a s a dropdown.

**Comment by jekuer** [comment, marked as answer]
Have a look at https://add-to-calendar-button.com/advanced-use - it holds some more options to access the shadow dom via css. Check the option to go via ::parts. This way, you could sneak in your own icon.

**Reply by iamritiksharma** [reply]
@jekuer  Thanks! that worked.
